itest/containers/config.go

```diff
@@ -25,13 +22,17 @@ const (
 
 // NewImageConfig returns ImageConfig needed for running e2e test.
 func NewImageConfig(t *testing.T) ImageConfig {
-	babylondVersion, err := testutil.GetBabylonVersion()
-	require.NoError(t, err)
+	// TODO: when new docker hub of babylon is released with
```

Comment:
maybe for now we can copy / paste `ParseV0StakingTxWithoutTag` to some separate file, and switch it later after version bump ? ( I do not like the those todos here 😅 )

Reply:
that is the weird part, the func `ParseV0StakingTxWithoutTag` is already on main
one image in docker hub was pushed https://github.com/babylonlabs-io/babylon/actions/runs/13767947156/job/38499612301

But if I let the Babylon version to load from the go.mod it doesn't find based on  `v1.99.0-snapshot.250211.0.20250310150510-16525b71a7eb`

Not sure how it was done before

Reply:
thinking right now it just clicked me, we had released a new tag 
![image](https://github.com/user-attachments/assets/a2d61552-21f4-4494-9c9d-05d347ad46d8)

Reply:
understood, modified to get the babylon from latest released tag on main v1.99.0-snapshot.250211